plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -742,6 +742,11 @@ private void addSplitPluginDependencies(String thisPlugin, MavenRunner.Config mc
             Map<String,VersionNumber> toReplace = new HashMap<String,VersionNumber>();
             Map<String,VersionNumber> toAddTest = new HashMap<String,VersionNumber>();
             Map<String,VersionNumber> toReplaceTest = new HashMap<String,VersionNumber>();
+            overridePlugins.forEach(plugin -> {
+                toReplace.put(plugin.getName(), plugin.getVersion());
```

Comment:
will it work correctly if the plugin is missing in pom.xml?

Reply:
Haven't check that usecase. To me, as it's `overridePlugins`, it was to set the version of a plugin which is already required by the one we are testing.

Reply:
As long as we modify the effective POM here, it should be fine